WooCommerce/Classes/Extensions/Array+Helpers.swift

```diff
@@ -13,6 +13,11 @@ extension Array {
 
         return removeFirst()
     }
+
+    /// A Boolean value indicating whether an array has characters.
+    public var isNotEmpty: Bool {
```

Comment:
I'm curious why this needs to be `public`. Removing it seems to work fine. 🤔 

```suggestion
var isNotEmpty: Bool {
```